templates/systemd-service_db.template

```diff
@@ -8,6 +8,7 @@ TimeoutStartSec=infinity
 TimeoutStopSec=15
 Restart=always
 RestartSec=2s
+
 ExecStartPre=-/usr/bin/docker stop {{ service_db_name }}
 ExecStartPre=-/usr/bin/docker rm {{ service_db_name }}
 ExecStartPre=/usr/bin/docker pull {{ docker_db_image }}
```

Comment:
Please create a variable for this command and make it work as in the main template (e.g., insert the line only if the variable is defined).